src/DssSpell.t.sol

```diff
@@ -904,63 +908,346 @@ contract DssSpellTest is DssSpellTestBase {
     }
 
     // SPELL-SPECIFIC TESTS GO BELOW
-    function testCronSequencerClipperMomJobReplaced() public {
-        address CRON_SEQUENCER = addr.addr("CRON_SEQUENCER");
-        address CRON_CLIPPER_MOM_JOB = 0xc3A76B34CFBdA7A3a5215629a0B937CBDEC7C71a;
-        address CRON_CLIPPER_MOM_JOB_NEW = 0x7E93C4f61C8E8874e7366cDbfeFF934Ed089f9fF;
+    bytes32           constant  SRC_ILK       = "PSM-USDC-A";
+    bytes32           constant  DST_ILK       = "LITE-PSM-USDC-A";
+    PsmAbstract       immutable srcPsm        = PsmAbstract(      addr.addr("MCD_PSM_USDC_A"));
+    LitePsmLike       immutable dstPsm        = LitePsmLike(      addr.addr("MCD_LITE_PSM_USDC_A"));
+    address           immutable pocket        =                   addr.addr("MCD_LITE_PSM_USDC_A_POCKET");
+    GemAbstract       immutable gem           = GemAbstract(      addr.addr("USDC"));
+    CronSequencerLike immutable sequencer     = CronSequencerLike(addr.addr("CRON_SEQUENCER"));
+    LitePsmJobLike    immutable litePsmJob    = LitePsmJobLike(  0x0C86162ba3E507592fC8282b07cF18c7F902C401);
+    uint256           constant  srcKeep       = 200_000_000 * WAD;
+    uint256           constant  srcTin        = 0.0001 ether;
+    uint256           constant  srcTout       = 0.0001 ether;
+    uint256           constant  srcMaxLine    = 2_500 * MILLION * RAD;
+    uint256           constant  srcGap        = 200 * MILLION * RAD;
+    uint256           constant  srcTtl        = 12 hours;
+    uint256           constant  dstBuf        = 200 * MILLION * WAD;
+    uint256           constant  dstMaxLine    = 7_500 * MILLION * RAD;
+    uint256           constant  dstGap        = 200 * MILLION * RAD;
+    uint256           constant  dstTtl        = 12 hours;
+    uint256           constant  rushThreshold = 20_000_000 * WAD;
+    uint256           constant  gushThreshold = 20_000_000 * WAD;
+    uint256           constant  cutThreshold  = 300_000 * WAD;
+
+    function testRwaConduitsPsmUpdate() public {
```

Comment:
We normally do sanity checks: checking expected values not only after the spell, but also before

Reply:
added sanity checks: [436a4af](https://github.com/makerdao/spells-mainnet/pull/422/commits/436a4afa72a7ae2a7a4a703286656451cda2c8c9)